BitwardenShared/Core/Auth/Models/Response/IdentityTokenErrorModel.swift

```diff
@@ -1,5 +1,13 @@
 import Foundation
 
+// MARK: - IdentityTokenErrors
+
+/// An error enumeration for `IdentityTokenRequest`.
```

Comment:
⛏️ I would change a bit the description given that the content itself is not an enumeration:
```suggestion
/// Constants for the `error` type returned in `IdentityTokenErrorModel`.
```